torch/_inductor/fx_passes/split_cat.py

```diff
@@ -454,10 +454,10 @@ def get_simplified_split_ranges(
             split_ranges,
         ):  # This need not be a strict condition
             # However, we keep it now for simplicity.
-            return
+            return # type: ignore[return-value]
```

Comment:
This does seem like an actual bug in our typing logic. Should be equivalent to returning None